test/cbmc/proofs/ARP/ARPGetCacheEntryByMac/ARPGetCacheEntryByMac_harness.c

```diff
@@ -22,14 +24,19 @@ const uint8_t ucMACAddress[ 6 ];
 void harness()
 {
     MACAddress_t xMACAddress;
-    uint32_t ulIPAddress;
+    uint32_t ulIPAddress, x;
     struct xNetworkEndPoint xLocalEndPoint;
-    struct xNetworkEndPoint * pxLocalEndPointPointer = &xLocalEndPoint;
+    struct xNetworkEndPoint * pxLocalEndPointPointer = nondet_bool() ? &xLocalEndPoint : NULL;
 
     /* Assume that the list of interfaces/endpoints is not initialized. */
     __CPROVER_assume( pxNetworkInterfaces == NULL );
     __CPROVER_assume( pxNetworkEndPoints == NULL );
 
+    for( x = 0; x < ipconfigARP_CACHE_ENTRIES; x++ )
+    {
+        xARPCache[ x ].pxEndPoint = nondet_bool() ? NULL : malloc( sizeof( NetworkEndPoint_t ) );
```

Comment:
CBMC malloc() can return NULL, as I recall.

Reply:
It can, but we have not switched to the flag that enables it since it made proofs slower - leading to slow PR check runs. Thus, we need to have that kind of statement to mimic the behavior of `mallocCanFail`.

Reply:
How much slower?

Reply:
At that time, the proofs were taking 16 minutes, and using the flag pushed that to 25 minutes.
That was very slow for a person waiting on the results of the PR checks. We decided that we rather not switch to that flag - at least in the IPv6 branch where the proofs are slower than the one in the IPv4 branch.